tests/deploy/k8s-cloudbeat-tests/values.yaml

```diff
@@ -14,6 +14,8 @@ image:
 testData:
```

Comment:
ES and Kibana versions are hardcoded to 8.5 here, do you want to update them? or get them dynamically?

Reply:
This is the default version. When executing `just` command `deploy-tests-helm` ES and Kibana versions are overwritten.